cve_bin_tool/output_engine/__init__.py

```diff
@@ -639,25 +648,20 @@ def generate_vex(self, all_cve_data: Dict[ProductInfo, CVEData], filename: str):
                     }
                 ]
                 vulnerability["ratings"] = ratings
-                vulnerability["cwes"] = []
                 vulnerability["description"] = cve.description
-                vulnerability["recommendation"] = ""
-                vulnerability["advisories"] = []
-                vulnerability["created"] = "NOT_KNOWN"
-                vulnerability["published"] = "NOT_KNOWN"
-                vulnerability["updated"] = "NOT_KNOWN"
+                vulnerability["updated"] = cve.last_modified
                 analysis = {
                     "state": analysis_state[cve.remarks],
                     "response": response_state[cve.remarks],
                     "justification": "",
                     "detail": cve.comments,
                 }
                 vulnerability["analysis"] = analysis
-                bom_urn = "NOTKNOWN"
                 bom_version = 1
+                # cve-bin-tool specific reference string to include vendor information
                 vulnerability["affects"] = [
                     {
-                        "ref": f"urn:cdx:{bom_urn}/{bom_version}#{product_info.product}-{product_info.version}",
+                        "ref": f"urn:cbt:{bom_version}/{product_info.vendor}#{product_info.product}-{product_info.version}",
```

Comment:
I assume `cbt` is a 'fantasy' URN Namespace for cve-bin-tool?

Indeed makes sense to have something like this here - it seems like a lot of work still needs to be done in the wider ecosystem around identification of software :)

Reply:
@raboof. Exactly. cbt stands for cve-bin-tool in case you haven't worked that out! I assume cdx stands for CycloneDX.